scripts/build/artifactory.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+#to be executed from the root project directory
+git rev-parse --short HEAD > version.txt
```

Comment:
> To make the solution more bullet-proof, it would be nice to have an information in the version.txt file if the source was changed locally with respect to HEAD. This could look like adding [locally modified] string after the version hash. In order to implement this feature, this or that SO answer might be helpful.
> 
> I'd go with:
> [[ -z $(git status -s) ]]
> as it is simple and does what we need.

Ok